roottest/root/multicore/CMakeLists.txt

```diff
@@ -127,10 +127,12 @@ ROOTTEST_ADD_TEST(exectsautoparse
                   MACRO exectsautoparse.C
                   OUTREF exectsautoparse.ref)
 
-ROOTTEST_ADD_TEST(exectsinclude
-                 MACRO  exectsinclude.C
-                 OUTCNVCMD grep -v -e "Line processed"
-                 OUTREF exectsinclude.ref)
+if(geom)
+  ROOTTEST_ADD_TEST(exectsinclude
```

Comment:
This test does not fundamentally depends on geom and already has internal code to support other optional feature missing.  How does it fail without geom?